test/TestProjects/MgmtKeyvault/tests/Generated/Mock/VaultCollectionTest.cs

```diff
@@ -57,13 +95,16 @@ public async Task CreateOrUpdate2()
                     DefaultAction = new MgmtKeyvault.Models.NetworkRuleAction("Deny"),
                 },
             });
+            parameters.Properties.NetworkAcls.IpRules.Add(new MgmtKeyvault.Models.IPRule(value: "124.56.78.91"));
+            parameters.Properties.NetworkAcls.IpRules.Add(new MgmtKeyvault.Models.IPRule(value: "'10.91.4.0/24'"));
```

Comment:
this single quote does not look correct? Could you please double confirm?

Reply:
comes from example files: https://github.com/Azure/azure-rest-api-specs/blob/a33fb4495e4c51cff80c76432bd5ea88e8088de9/specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-06-01-preview/examples/createVaultWithNetworkAcls.json#L23